README.md

```diff
@@ -1,7 +1,21 @@
 # create-origami-component
 
+A command line tool for creating new Origami components.
+
 ## Usage
 
 ```shell
 npm init origami-component
 ```
+
+## Contact
+
+If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```

Comment:
```suggestion
If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/.../issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```

Reply:
the link had o-subs-card, I replaced with ... as I have no idea what to replace with 🙈

Reply:
sorry I spotted and already resolved it, thanks for flagging!